contrib/vmstat.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env drgn
+# Copyright (c) SUSE Linux.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""Dump /proc/vmstat statistics.
+
+   Skip the last enumerator item as it holds the number of enumrator values."""
+
+from drgn.helpers.linux.cpumask import for_each_online_cpu
+from drgn.helpers.linux.percpu import per_cpu
+
+
+def print_event_line(event, counter):
+    print(f"{event.name:<36} {counter.value_():>16}")
+
+
+print(f"{'Event':<36} {'Count':>16}")
+
+# 1) vm_zone_stat statistics
+print("VM_ZONE_STAT:")
+vm_zone_stat = prog["vm_zone_stat"]
+for event in prog.type("enum zone_stat_item").enumerators[:-1]:
+    print_event_line(event, vm_zone_stat[event.value].counter)
+print()
+
+# 2) vm_node_stat statistics are there since v5.14
+if "vm_numa_event" in prog:
```

Comment:
I think this check might be switched around. The body of this if statement uses `vm_node_stat`, which was added in https://github.com/torvalds/linux/commit/75ef7184053989118d3814c558a9af62e7376a58 in v4.8.

```suggestion
# 2) vm_node_stat statistics are there since v4.8
if "vm_node_stat" in prog:
```